server/events/yaml/raw/global_cfg.go

```diff
@@ -163,6 +179,13 @@ func (r Repo) ToValid(workflows map[string]valid.Workflow) valid.Repo {
 		id = r.ID
 	}
 
+	var branchRegex *regexp.Regexp
+	if r.HasRegexBranch() {
+		withoutSlashes := r.Branch[1 : len(r.Branch)-1]
+		// checked in repo Validate()
```

Comment:
NIT: Please consider using sentece simillar to the comment above.

Keeping the same pattern makes it easier to read and laso from this it is not crystal clear for me what and for what reason was checked.

Reply:
updated